### DISCUSSION TITLE
I can't download anymore

### DISCUSSION THREAD

**Discussion by Kingscity01** [TeamNewPipe/NewPipe#11653] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

I can't download music and movie anymore  using new pipe

**Comment by asandikci** [comment]
I can still download. Could you please give more specific information about your problem? Which video are you trying to download? Resolution? Video/music/caption? Using VPN?
